t/cobrand/bromley.t

```diff
@@ -1775,6 +1778,83 @@ subtest 'check direct debit reconcilliation' => sub {
     is $failed_new_sub->state, 'hidden', 'failed sub still hidden on second run';
     $ad_hoc_skipped->discard_changes;
     is $ad_hoc_skipped->state, 'unconfirmed', "ad hoc report not confirmed on second run";
+
+};
+
+subtest 'Garden Waste new subs alert update emails contain bin collection days link' => sub {
+    FixMyStreet::override_config {
+        ALLOWED_COBRANDS => 'bromley',
+    }, sub {
+        $mech->clear_emails_ok;
+        my $contact = $mech->create_contact_ok(body => $body, category => 'Garden Subscription', email => 'garden@example.com');
+
+        my $new_sub = ($mech->create_problems_for_body( 1, $body->id, 'Garden Subscription - New', {
+            category => 'Garden Subscription',
+            latitude => 51.402096,
+            longitude => 0.015784,
+            cobrand => 'bromley',
+            cobrand_data => 'waste',
+            areas => '8141',
+            user => $user,
+            send_method_used => 'Open311',
+            whensent => 'now()',
+            external_id => '456',
+            extra => {
+                contributed_by => $staffuser->id,
+                Subscription_Type => 1,
+                Subscription_Details_Quantity => 1,
+                payment_method => 'direct_debit',
+                property_id => '54323',
+                uprn => '654321',
+            },
+        }))[0];
+
+        my $update = FixMyStreet::DB->resultset('Comment')->find_or_create({
+            problem_state => 'action scheduled',
+            problem_id => $new_sub->id,
+            user_id    => $staffuser->id,
+            name       => 'Staff User',
+            mark_fixed => 'f',
+            text       => "Green bin on way",
+            state      => 'confirmed',
+            confirmed  => 'now()',
+            anonymous  => 'f',
+        });
+
+        my $alert = FixMyStreet::DB->resultset('Alert')->create({
+            user => $user,
+            parameter => $new_sub->id,
+            alert_type => 'new_updates',
+            whensubscribed => '2021-09-27 12:00:00',
+            cobrand => 'bromley',
+            cobrand_data => 'waste',
+        });
+        $alert->confirm;
+
+        FixMyStreet::Script::Alerts::send();
+
+        my $alerts = FixMyStreet::App->model('DB::Alert')->search({
+            alert_type => 'new_updates',
+            parameter => $new_sub->id,
+        });
+        is $alerts->count, 1, "alert found for GGW update";
+
+        $mech->email_count_is(1, 'GGW update alert email found');
```

Comment:
This line, the line below, and the for loop can be replaced (the for loop itself, I mean, not its contents) as `my $email = $mech->get_email` - when used in a scalar context, it checks for 1 email and returns it directly.

Reply:
What was I thinking?